exersice1.js

```diff
@@ -0,0 +1,100 @@
+let yearFuture = 2051;
+let age;
+
+age = yearFuture - yearOfBirth;
+
+console.log(`You will be ${age} old in ${yearFuture}`);
+
+// Goodboy-Oldboy (A dog age calculator)
+// The same friend(who by the way loves dogs) asks how old his 
+// dog will be in 2045. Hmm you think, lets make this into a
+// product as well! Dogs age can either be measured in dog years
+// or in human years, this we want to take into consideration!
+
+const dogYearOfBirth = 2022;
+const dogYearFuture = 2025;
+let dogYear;
+
+dogYear = dogYearFuture - dogYearOfBirth;
+
+const shouldShowResultInDogYears = true;
+
+if (shouldShowResultInDogYears === false) {
+  console.log(`Your dog will be ${dogYear * 7} dog years old in ${dogYear}`);
+} else {
+  console.log(`Your dog will be ${dogYear} human years old in ${dogYear}`);
+}
+
+
+// Housey pricey (A house price estimator)
+// Two of your friends are considering buying a house, but cannot 
+// figure out what the right price should be. Your friends know the
+// width, the height and the depth of the house and the garden
+// size.Lets help them figure out if they paid too much:
+
+const widthHouse = 8;
+const heightHouse = 10;
+const depthHouse = 10;
+const gardenSize = 100;
+
+//housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const peterHouseParameters = {width: 8, heigth: 10, deep: 10, garden: 100, price: 2500000}
+let housePricePeter = peterHouseParameters.width * peterHouseParameters.heigth* peterHouseParameters.deep * 2.5 * 1000 + peterHouseParameters.garden * 300;
+console.log("house Price Peter", housePricePeter);
+if (housePricePeter > peterHouseParameters.price) {
+  console.log('Price too hight');
+} else {
+  console.log("Price is lower");
+}
+
+
+const juliaHouseParameters = {
+  width: 5,
+  heigth: 11,
+  deep: 8,
+  garden: 70,
+  price: 1000000,
+};
+let housePriceJulia =
+  juliaHouseParameters.width * juliaHouseParameters.heigth * juliaHouseParameters.deep * 2.5 * 1000 +
+  juliaHouseParameters.garden * 300;
+console.log("house Price Julia", housePriceJulia);
+
+if (housePriceJulia > juliaHouseParameters.price) {
+  console.log("Price too hight");
+} else {
+  console.log("Price is lower");
+}
+
+// Ez Namey (Startup name generator) Optional
+// At a meetup you overhear a conversation between two developers. 
+// It went something like this: "Man i wish picking a startup name 
+// was easier! You need to be creative, funny and interesting, its 
+// nearly impossible!" Another problem to solve, awesome!
+
+const firstWords = ['Cool', 'Great', 'Maxi', 'Big', 'Digital', 'Awesome', 'Smart', 'Super'];
+const secondWords = [
+  "Corporate",
+  "Businness",
+  "Journey",
+  "Cooperation",
+  "Travel",
+  "Corporation",
+  "Sience",
+  "Steps",
+  "Alpha",
+  "Comp",
+];
+
+const randomNumber = Math.floor(Math.random() * 10);
+const newName = firstWords[randomNumber] + ' ' + secondWords[randomNumber];
+console.log(`The startup: "${newName}" contains ${newName.length} characters`)
```

Comment:
Nice! Just one thing, when you use ﻿`newName.length` it also counts the part ﻿` + ' ' +` in the length. If you want to count just the number of the letters, you should count them separately like this:

```
const firstWordCount = firstWords[randomNumber].length
const secondWordCount = secondWords[randomNumber].length
const numberOfLetters = firstWordCount + secondWordCount

console.log(`The startup: "${newName}" contains ${numberOfLetters} characters`)
```

or you can simply subtract 1 from the sum :D 
`
const numberOfLetters = newName.length - 1
`

Reply:
You're right, but in our hometask  was an example that counts all symbols in new name.
  "An example could be: "The startup: "Easy Corporation" contains 16 characters" "

Reply:
ok I see, in that case, all good:)